OpenOversight/app/models.py

```diff
@@ -39,6 +42,27 @@
 )
 
 
+date_updated_cache = TTLCache(maxsize=1024, ttl=60)
```

Comment:
I forgot to update the ttl but does a 12 hour ttl make sense?

Reply:
Yes I think so!